main.js

```diff
@@ -0,0 +1,138 @@
+const alphabet = ["A", "B", "C", "D", "E", "F", "G", "H", "I", "J", "K", "L", "M", "N", "O", "P", "Q", "R", "S", "T", "U", "V", "W", "X", "Y", "Z"]
+const gallowImages = ["img/1.jpg", "img/2.jpg", "img/3.jpg", "img/4.jpg", "img/5.jpg", "img/6.jpg", "img/7.jpg", "img/8.jpg", "https://t3.ftcdn.net/jpg/01/84/91/74/360_F_184917449_2UTmC5D609xMxaAyqGDzfNUt64CnOdFG.jpg", "https://fsb.zobj.net/crop.php?r=yDx9wB7yqfbxfzAok8T8bXK7tvVg7P24mhDNXvW32TKa1Vt9giET2X0_OQCBlI4gVTXFM3ShDs4xziVel1j0l1R1UY-xbs-wihPXPU4QW7FIGBvrBBGnkf-U-tn5oEylzKRJLNhcVeUBr3jisoCJOT5dOKKISBC2gAJznCClBiX2REVOTkQbbHtVBug"];
+let encryptedWord = [];
+let startPage = document.getElementById("word-input");
+let hangmanPage = document.getElementById("word-guessing");
+let wordToGuess;
+let mismatchedLetters = 0;
+let completeBoard = encrypted;
+let alphabetButtons = letters;
+let gallowsImages = images;
+let resetButton = restart;
```

Comment:
Can you get rid of some global variables and instead send the as parameters where needed?